modules/subchannel/include/postprocessors/SCMDuctHeatPostprocessor.h

```diff
@@ -0,0 +1,34 @@
+//* This file is part of the MOOSE framework
+//* https://mooseframework.inl.gov
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "GeneralPostprocessor.h"
+#include "SubChannelMesh.h"
+
+/**
+ * Calculates the total heat $[W]$ that goes in_to/out_of the coolant through the duct based on
```

Comment:
```suggestion
 * Calculates the total heat flow $[W]$ that goes in the coolant through the duct based on
```

either in or out. the sign matters here